css-position-4/Overview.bs

```diff
@@ -353,10 +355,13 @@ Top Layer Manipulation {#top-manip}
 	To <dfn export>process top layer removals</dfn>,
 	given a {{Document}} |doc|:
 
-	1. For each element |el| in |doc|'s [=pending top layer removals=]:
-		if |el|&apos;s computed value of 'overlay' is ''none'',
-		or |el| is [=not rendered=],
-		[=remove from the top layer immediately=] |el|.
+	1. For each (element |el| and steps |afterRemoval|) in |doc|'s
+		[=pending top layer removals=]: if |el|&apos;s computed value of
+		'overlay' is ''none'', or |el| is [=not rendered=]:
+
+		1. [=remove from the top layer immediately=] |el|.
```

Comment:
This should just pass after removal through as they're called in that algorithm.

Reply:
Good point.